src/content/docs/develop/state-management.mdx

```diff
@@ -2,32 +2,180 @@
 title: State Management
 sidebar:
   order: 1
-  badge:
-    text: Stub
-    variant: caution
 ---
 
-import Stub from '@components/Stub.astro';
+In a Tauri application, you often need to keep track of the current state of your application or manage the lifecycle of things associated with it. Tauri provides an easy way to manage the state of your application using the `Manager` API, and read it when commands are called.
 
-<Stub>
+Here is a simple example:
 
-Define a state struct and let Tauri manage it:
+```rust
+use tauri::{Builder, Manager};
+
+struct AppData {
+  welcome_message: &'static str,
+}
+
+// In `fn main()`:
+Builder::default()
+  .setup(|app| {
+    app.manage(AppData {
+      welcome_message: "Welcome to Tauri!",
+    });
+    Ok(())
+  })
+  .run(tauri::generate_context!())
+  .unwrap();
+```
+
+You can later access your state via the `App` instance:
```

Comment:
This is partially true, you can access the state through any type that implements the `Manager` trait